homeassistant/components/airq/number.py

```diff
@@ -0,0 +1,80 @@
+"""Definition of air-Q number platform used to control the LED strips."""
+
+from __future__ import annotations
+
+import logging
+
+from aioairq import AirQ
+
+from homeassistant.components.number import NumberEntity, NumberEntityDescription
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import AirQConfigEntry, AirQCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+LED_VALUE_DEFAULT = 6.0
+
+
+AIRQ_LED_BRIGHTNESS = NumberEntityDescription(
+    key="airq_led_brightness",
+    translation_key="airq_led_brightness",
+    native_min_value=0.0,
+    native_max_value=10.0,
+    native_step=0.1,
```

Comment:
why don't we do 0-100 with a one step? That sounds more logical and then we can use PERCENTAGE as unit

Reply:
Fair point. The last commit I just pushed handles the conversion between the percentage and the [0, 10] scale expected by the firmware/`aioairq`. 
I will consider now if it makes sense to push this into `aioairq` and further bump the dependency...

Reply:
Thanks for looking into it all!

Reply:
@joostlek I did push these changes to `aioairq`, it now handles the brightness in percent by default. I reset and the aforementioned commit and force-pushed a new one.